terraform/variables.tf

```diff
@@ -1,34 +1,34 @@
-variable "client_id" {}
-variable "client_secret" {}
-variable "tenant_id" {}
-variable "rbac_server_app_id" {}
-variable "rbac_server_app_secret" {}
-variable "rbac_client_app_id" {}
-
-variable "kubernetes_version" {
-    default = "1.11.3"
-}
-
-variable "agent_count" {
-    default = 1
-}
-
-variable "ssh_public_key" {
-    default = "~/.ssh/id_rsa.pub"
-}
-
-variable "dns_prefix" {
-    default = "azadk8s2"
-}
-
-variable cluster_name {
-    default = "azadk8s2"
-}
-
-variable resource_group_name {
-    default = "azure-ad-k8s2-rg"
-}
-
-variable location {
-    default = "West Europe"
+variable "client_id" {}
+variable "client_secret" {}
+variable "tenant_id" {}
+variable "rbac_server_app_id" {}
+variable "rbac_server_app_secret" {}
+variable "rbac_client_app_id" {}
+
+variable "kubernetes_version" {
+  default = "1.11.5"
```

Comment:
It can be problably now bumped to `1.13.7`:
```
$ az aks get-versions --location westeurope --output table
WARNING: The behavior of this command has been altered by the following extension: aks-preview
KubernetesVersion    Upgrades
-------------------  ----------------------------------------
1.14.3(preview)      None available
1.14.1(preview)      1.14.3(preview)
1.13.7               1.14.1(preview), 1.14.3(preview)
1.13.5               1.13.7, 1.14.1(preview), 1.14.3(preview)
1.12.8               1.13.5, 1.13.7
1.12.7               1.12.8, 1.13.5, 1.13.7
1.11.10              1.12.7, 1.12.8
1.11.9               1.11.10, 1.12.7, 1.12.8
1.10.13              1.11.9, 1.11.10
1.10.12              1.10.13, 1.11.9, 1.11.10
```